### DISCUSSION TITLE
什么时候添加chatgpt-4o-latest呢，还有0806版本

### DISCUSSION THREAD

**Discussion by mahaonanStart** [BurnHair/docs#186] [功能建议]
什么时候添加chatgpt-4o-latest呢，还有0806版本。上好久了

**Comment by GrayXu** [comment]
这明显都是azure openai来的模型，azure还没上0806呢

**Reply by gengjiawen** [reply]
0806有了

**Reply by GrayXu** [reply]
> 0806有了

yes 现在azure上0806了

**Comment by BennyThink** [comment]
https://api.burn.hair/ 这里有了 :-) 可以先凑合用
